Last-Update: 2014-10-06
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS in antlr.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -225,9 +225,9 @@
   set(ANTLR3C_LIBRARIES "-L${PROJECT_BINARY_DIR}/antlr-wb -lantlr3c_wb")
 
   ExternalProject_Add(antlr-wb
     SOURCE_DIR ${PROJECT_SOURCE_DIR}/ext/antlr-runtime
-    CONFIGURE_COMMAND autoreconf -i -s && ${PROJECT_SOURCE_DIR}/ext/antlr-runtime/configure
+    CONFIGURE_COMMAND ${PROJECT_SOURCE_DIR}/ext/antlr-runtime/configure
         --prefix=${PROJECT_BINARY_DIR}/antlr-wb
         --libdir=${PROJECT_BINARY_DIR}/antlr-wb
 	--disable-abiflags
     BUILD_COMMAND make
