Last-Update: 2012-02-20
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: no
Description: fix FTBFS with automake 1.11.2


--- a/library/forms/swig/Makefile.am
+++ b/library/forms/swig/Makefile.am
@@ -1,8 +1,8 @@
 
 pkglib_LTLIBRARIES=_mforms.la
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
-pkglib_DATA=mforms.py
+pkgmodulesdir=$(libdir)/@PACKAGE@/modules
+pkgmodules_DATA=mforms.py
 
 _mforms_la_SOURCES=\
 mforms_wrap.cxx
--- a/modules/wb.utils/Makefile.am
+++ b/modules/wb.utils/Makefile.am
@@ -1,6 +1,6 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgmodulesdir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=catalog_utils.grt.lua  tools.grt.lua  wb_utils_grt.py sqlide_grt.py text_grt.py sql_reformatter.py\
+pkgmodules_DATA=catalog_utils.grt.lua  tools.grt.lua  wb_utils_grt.py sqlide_grt.py text_grt.py sql_reformatter.py\
             wb_dev_utils_grt.py table_utils_grt.py code_utils_grt.py
 
--- a/plugins/wb.admin/frontend/Makefile.am
+++ b/plugins/wb.admin/frontend/Makefile.am
@@ -1,7 +1,7 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgmodulesdir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=\
+pkgmodules_DATA=\
     wb_admin_grt.py\
     wb_admin_monitor.py\
     wb_admin_utils.py\
--- a/plugins/wb.doclib/Makefile.am
+++ b/plugins/wb.doclib/Makefile.am
@@ -1,7 +1,7 @@
 
-pkglibdir=$(libdir)/@PACKAGE@/modules
+pkgmodulesdir=$(libdir)/@PACKAGE@/modules
 
-pkglib_DATA=\
+pkgmodules_DATA=\
   mysqldoclib.py\
   wb_doclib_grt.py
 
