Last-Update: 2012-10-10
Forwarded: not-needed,backported
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=682691
Description: fix application freeze on exit (RC)

--- a/library/grt/src/grtpp_undo_manager.cpp
+++ b/library/grt/src/grtpp_undo_manager.cpp
@@ -627,11 +627,11 @@
 
 
 UndoManager::~UndoManager()
 {
-  g_static_rec_mutex_free(&_mutex);
   _changed_signal.disconnect_all_slots(); // prevent emission in reset()
   reset();
+  g_static_rec_mutex_free(&_mutex);
 }
 
 
 
