Last-Update: 2013-06-04
Forwarded: no
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=710627
Description: use `python-config` to get libpython flags

--- a/configure.in
+++ b/configure.in
@@ -242,17 +242,10 @@
 AC_CHECK_PROG(PYTHON26, python2.6, [yes])
 AC_CHECK_PROG(PYTHON27, python2.7, [yes])
 # Python headers
 AC_MSG_CHECKING(for Python libraries)
-PYTHON_LIBS=[$(python -c "from distutils import sysconfig
-import os 
-cfg=sysconfig.get_config_vars()
-lib = os.path.join(cfg['LIBDIR'], cfg['LDLIBRARY'])
-if os.path.exists(lib):
-    print \"-L\" + cfg['LIBDIR'] + \" -l\" + cfg['LDLIBRARY'].lstrip(\"lib\").rstrip(\".so\")")]
-PYTHON_CFLAGS=[$(python -c "from distutils import sysconfig
-print sysconfig.get_python_inc()")]
-PYTHON_CFLAGS="-I$PYTHON_CFLAGS"
+PYTHON_LIBS=[$(python-config --libs)]
+PYTHON_CFLAGS=[$(python-config --cflags)]
 if test "$PYTHON_LIBS" = ""; then
   AC_MSG_RESULT(not found)
   AC_MSG_ERROR([Python C development headers/libraries not found])
 else
