Last-Update: 2016-11-13
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS with MariaDB.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -104,9 +104,9 @@
 
 FIND_PROGRAM(PYTHON_EXEC "python2")
 include(FindPackageHandleStandardArgs)
 
-find_package(MySQL 5.6.0 REQUIRED)
+find_package(MySQL REQUIRED)
 if (UNIX)
 	set(MYSQL_DEFINITIONS "-DBIG_JOINS=1"  "-fno-strict-aliasing")
 endif()
 
