Last-Update: 2016-11-13
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS with MariaDB.

--- a/plugins/migration/copytable/copytable.cpp
+++ b/plugins/migration/copytable/copytable.cpp
@@ -78,9 +78,9 @@
     case MYSQL_TYPE_BLOB: return "MYSQL_TYPE_BLOB";
     case MYSQL_TYPE_VAR_STRING: return "MYSQL_TYPE_VAR_STRING";
     case MYSQL_TYPE_STRING: return "MYSQL_TYPE_STRING";
     case MYSQL_TYPE_GEOMETRY: return "MYSQL_TYPE_GEOMETRY";
-    case MYSQL_TYPE_JSON: return "MYSQL_TYPE_JSON";
+//    case MYSQL_TYPE_JSON: return "MYSQL_TYPE_JSON";
     default:
       return "UNKNOWN";
   }
 }
@@ -240,9 +240,9 @@
       case MYSQL_TYPE_NEWDECIMAL:
       case MYSQL_TYPE_STRING:
       case MYSQL_TYPE_VAR_STRING:
       case MYSQL_TYPE_BIT:
-      case MYSQL_TYPE_JSON:
+//      case MYSQL_TYPE_JSON:
         if (!col->is_long_data)
           bind.buffer_length = (unsigned)col->source_length+1;
 
         bind.length = (unsigned long*)malloc(sizeof(unsigned long));
@@ -1550,10 +1550,10 @@
             rowbuffer[index].buffer_type == MYSQL_TYPE_MEDIUM_BLOB ||
             rowbuffer[index].buffer_type == MYSQL_TYPE_LONG_BLOB ||
             rowbuffer[index].buffer_type == MYSQL_TYPE_BLOB ||
             rowbuffer[index].buffer_type == MYSQL_TYPE_STRING ||
-            rowbuffer[index].buffer_type == MYSQL_TYPE_GEOMETRY ||
-            rowbuffer[index].buffer_type == MYSQL_TYPE_JSON)
+            rowbuffer[index].buffer_type == MYSQL_TYPE_GEOMETRY)
+//            rowbuffer[index].buffer_type == MYSQL_TYPE_JSON)
           {
             if (rowbuffer[index].buffer_length)
               free(rowbuffer[index].buffer);
 
@@ -2356,9 +2356,9 @@
     case MYSQL_TYPE_VARCHAR:
     case MYSQL_TYPE_STRING:
     case MYSQL_TYPE_ENUM:
     case MYSQL_TYPE_SET:
-    case MYSQL_TYPE_JSON:
+//    case MYSQL_TYPE_JSON:
       _bulk_insert_record.append("'", 1);
       ret_val = _bulk_insert_record.append_escaped((char*)(*_row_buffer)[col_index].buffer, *(*_row_buffer)[col_index].length);
       _bulk_insert_record.append("'", 1);
       break;
--- a/backend/wbpublic/objimpl/db.query/db_query_Resultset.cpp
+++ b/backend/wbpublic/objimpl/db.query/db_query_Resultset.cpp
@@ -299,11 +299,11 @@
         case sql::DataType::ENUM:
         case sql::DataType::SET:
           type= "string";
           break;
-        case sql::DataType::JSON:
-          type = "json";
-          break;
+//        case sql::DataType::JSON:
+//          type = "json";
+//          break;
         case sql::DataType::SQLNULL:
           type = "null";
           break;          
       }
