Last-Update: 2013-06-19
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: correct path to ifconfig

--- a/plugins/wb.admin/backend/wb_server_control.py
+++ b/plugins/wb.admin/backend/wb_server_control.py
@@ -33,9 +33,9 @@
 import socket
 
 #-------------------------------------------------------------------------------
 def get_local_ip_list():
-    cmd    = "/bin/sh -c ifconfig"
+    cmd    = "/bin/sh -c /sbin/ifconfig"
     regexp = "inet addr:([0-9a-f:\.]+)"
 
     if hasattr(sys, 'getwindowsversion'):
         cmd    = 'ipconfig'
