Last-Update: 2019-08-02
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: fix FTBFS with MariaDB.

--- a/plugins/migration/copytable/copytable.cpp
+++ b/plugins/migration/copytable/copytable.cpp
@@ -2271,9 +2271,9 @@
       case MYSQL_TYPE_DATETIME2:
       case MYSQL_TYPE_TIME2:
 #endif
 #if MYSQL_VERSION_ID > 80016
-      case MYSQL_TYPE_TYPED_ARRAY: /* Used only for replication. */
+//      case MYSQL_TYPE_TYPED_ARRAY: /* Used only for replication. */
 #endif
         // TODO: implement handling
         break;
       case MYSQL_TYPE_GEOMETRY:
