Last-Update: 2014-04-02
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=739515
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: set relative RPATH for `wbcopytables` executable.

--- a/plugins/migration/CMakeLists.txt
+++ b/plugins/migration/CMakeLists.txt
@@ -13,8 +13,9 @@
     copytable/main.cpp
     copytable/converter.cpp
 )
 
+set_target_properties(wbcopytables PROPERTIES INSTALL_RPATH "$ORIGIN")
 target_link_libraries(wbcopytables wbbase ${MYSQL_LIBRARIES} ${ODBC_LIBRARIES} ${PCRE_LIBRARIES} ${PYTHON_LIBRARIES})
 
 install(TARGETS wbcopytables DESTINATION ${WB_INSTALL_BIN_DIR})
 
