Last-Update: 2015-12-22
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=739515
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: set relative RPATH for `wbcopytables` executable.

--- a/plugins/migration/CMakeLists.txt
+++ b/plugins/migration/CMakeLists.txt
@@ -87,8 +87,9 @@
       ${LIBZIP_LIBRARIES}
   )
    
    if(BUILD_FOR_GCOV)
+    set_target_properties(wbcopytables PROPERTIES INSTALL_RPATH "$ORIGIN")
     target_link_libraries(wbcopytables PRIVATE gcov)
    endif()
    if(COMMAND cotire)
        set_target_properties(wbcopytables PROPERTIES
