Last-Update: 2015-12-22
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=739515
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: set relative RPATH for `wbcopytables` executable.

--- a/plugins/migration/CMakeLists.txt
+++ b/plugins/migration/CMakeLists.txt
@@ -53,8 +53,9 @@
   else()
     target_compile_options(wbcopytables PUBLIC -fPIE -pie)
   endif()
 
+   set_target_properties(wbcopytables PROPERTIES INSTALL_RPATH "$ORIGIN")
    target_link_libraries(wbcopytables wbbase ${MySQL_LIBRARIES} ${ODBC_LIBRARIES} ${PCRE_LIBRARIES} ${PYTHON_LIBRARIES})
    if(BUILD_FOR_TESTS)
     target_link_libraries(wbcopytables gcov)
    endif()
